File: Python/2942.py
# 2942. Find Words Containing Character
# https://leetcode.com/problems/find-words-containing-character/

class Solution:
  def findWordsContaining(self, words: List[str], x: str) -> List[int]:
    out = []

    for (i, w) in enumerate(words):
      if (w.find(x) > -1):
        out.append(i)

    return out

File: Rust/2942.rs
/**
 * 2942. Find Words Containing Character
 * https://leetcode.com/problems/find-words-containing-character/
 */

impl Solution {
  pub fn find_words_containing(words: Vec<String>, x: char) -> Vec<i32> {
    let mut out: Vec<i32> = Vec::new();

    for (i, w) in words.iter().enumerate() {
      if w.find(x).is_some() {
        out.push(i as i32);
      }
    }

    return out;
  }
}
